src/Appwrite/Utopia/Database/Validator/Query/Filter.php

```diff
@@ -92,6 +92,8 @@ public function isValid($query): bool
         $attribute = $query->getAttribute();
 
         switch ($method) {
+            case Query::TYPE_SLEEP:
```

Comment:
Same here, this operator shouldn't be used by users, only internally.

Reply:
removed!